### DISCUSSION TITLE
guide/sponsor

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#375] [Announcements]
# guide/sponsor

If you think this project is helpful to you, you may wish to consider buying me a cup of milk tea😄. Buy AList Desktop: https://r.nn.ci/ad; Buy me a coffee: https://ko-fi.com/xh...

https://alist.nn.ci/guide/sponsor.html

**Comment by FabihaBaba** [comment]
https://cdn.nagordola.com.bd/



Hello 
I am interested to buy this script from you please contact me whatsapp  https://wa.me/+8801889248542

**Reply by FabihaBaba** [reply]
Or Email: bdmusicorg@gmail.com
